discovery-provider/Dockerfile.acdc

```diff
@@ -0,0 +1,3 @@
+FROM nethermind/nethermind:1.14.3
+
+RUN apt-get update && apt-get -y install curl jq
```

Comment:
If we want a more complex health check i'd propose we install either node or python here so we have more flexibility? thoughts @isaacsolo? don't want the image too big though.

Reply:
sure let's just leave it as is for now and try to add any necessary logic to the regular health_check